timemachine/fe/utils.py

```diff
@@ -271,6 +271,11 @@ def get_romol_conf(mol):
     return guest_conf / 10  # from angstroms to nm
 
 
+def get_mol_masses(mol):
```

Comment:
Nice! A quick search for `.GetMass()` in the project turns up ~ a dozen repetitions that could be replaced by this

Reply:
I'll leave this to a separate case: https://github.com/proteneer/timemachine/issues/703